sdk/core/azure-core/vcpkg.json

```diff
@@ -35,7 +35,8 @@
           "default-features": false,
           "features": [
             "ssl"
-          ]
+          ],
+          "version>=": "7.44"
```

Comment:
If you use `version>=` you need to specify a baseline, I believe.

Reply:
But I think we do - that pinned commit SHA that we have?

Reply:
We have builtin-baseline in our root vcpkg.json. If it was not in effect, the vcpkg step would fail, but the CI passes so it means it is read from there.